modtools/create-item.lua

```diff
@@ -44,8 +44,8 @@ local validArgs = utils.invert({
  'item',
 -- 'creature',
 -- 'caste',
- 'leftHand',
```

Comment:
This will break for anyone using these arguments - in the interests of compatibility, it might be best to keep them. I don't think having support for both will break anything from a quick glance through.

Reply:
readded